components/ReadSpeaker/src/index.scss

```diff
@@ -0,0 +1,1344 @@
+    );
+
+    &:hover {
+      --denhaag-readspeaker-switch-background-color: var(--denhaag-readspeaker-switch-active-hover-background-color);
+    }
+
+    &:focus-visible {
+      --denhaag-readspeaker-switch-background-color: var(--denhaag-readspeaker-switch-active-focus-background-color);
+    }
+
+    &:disabled,
+    &.disabled {
+      --denhaag-readspeaker-switch-background-color: var(--denhaag-readspeaker-switch-active-disabled-background-color);
+    }
+  }
+}
+
+#rs-dialog,
+#rs-dialog-fullscreen {
+  nav {
+    button {
+      @each $icon,
+        $path
+          in (
+            "play": var(--denhaag-readspeaker-icons-play),
+            "pause": var(--denhaag-readspeaker-icons-pause),
+            "speed": var(--denhaag-readspeaker-icons-speed),
+            "closer": var(--denhaag-readspeaker-icons-close),
+            "forward": var(--denhaag-readspeaker-icons-forward),
+            "rewind": var(--denhaag-readspeaker-icons-rewind),
+            "volume": var(--denhaag-readspeaker-icons-volume),
+            "stop": var(--denhaag-readspeaker-icons-stop),
+            "shift": var(--denhaag-readspeaker-icons-shift),
+            "control": var(--denhaag-readspeaker-icons-control),
+            "option": var(--denhaag-readspeaker-icons-option),
+            "help": var(--denhaag-readspeaker-icons-help),
+            "command": var(--denhaag-readspeaker-icons-command),
+            "font-minus": var(--denhaag-readspeaker-icons-font-minus),
+            "font-plus": var(--denhaag-readspeaker-icons-font-plus),
+            "font-settings": var(--denhaag-readspeaker-icons-font-settings),
+            "settings": var(--denhaag-readspeaker-icons-settings),
+            "pagemask": var(--denhaag-readspeaker-icons-pagemask),
+            "dictionary": var(--denhaag-readspeaker-icons-dictionary)
+          )
+      {
+        &.rsicn-#{$icon}::after {
+          @include readspeaker-icon-styles($path);
+        }
+      }
+
+      &.rsicn-play[data-rslang="title/arialabel:pause"]::after {
+        @include readspeaker-icon-styles(
+          'path("M9 6C9.55228 6 10 6.44772 10 7V17C10 17.5523 9.55228 18 9 18C8.44772 18 8 17.5523 8 17V7C8 6.44772 8.44772 6 9 6ZM15 6C15.5523 6 16 6.44772 16 7V17C16 17.5523 15.5523 18 15 18C14.4477 18 14 17.5523 14 17V7C14 6.44772 14.4477 6 15 6Z")'
+        );
+      }
+
+      &.rsicn-font-minus::after {
+        @include readspeaker-icon-styles(var(--denhaag-readspeaker-icons-font-minus));
+      }
+      &.rsicn-font-plus::after {
+        @include readspeaker-icon-styles(var(--denhaag-readspeaker-icons-font-plus));
+      }
+      &.rsicn-text-settings {
+        display: none !important;
+      }
+      &.rs-dialog-fullscreen::after {
+        @include readspeaker-icon-styles(var(--denhaag-readspeaker-icons-expand));
+      }
+      &.rsicn-cross::after {
+        @include readspeaker-icon-styles(var(--denhaag-readspeaker-icons-close));
+      }
+    }
+  }
+}
+
+// Dialog general settings
+dialog {
```

Comment:
No CSS selectors? This will be apply to all `<dialog>` right now. Or is it child of .denhaag-readspeaker? cant see properly with this indenting.